pythonforandroid/recipes/cryptography/__init__.py

```diff
@@ -1,27 +1,30 @@
 from pythonforandroid.recipe import CompiledComponentsPythonRecipe
-from os.path import dirname, join
+
 
 class CryptographyRecipe(CompiledComponentsPythonRecipe):
     name = 'cryptography'
-    version = '1.3'
-    url = 'https://github.com/pyca/cryptography/archive/{version}.tar.gz'
-    depends = [('python2', 'python3crystax'), 'openssl', 'idna', 'pyasn1', 'six', 'setuptools', 'enum34', 'ipaddress', 'cffi']
```

Comment:
Is there a particular reason why the cryptography recipe as it exists currently doesn't need all these hostpython recipes, but your one does? There's not necessarily any problem with it, just wondering what's different.